File: uri/1018.py
listinha = [100,50,20,10,5,2,1]

valor_entrada = int(input())

print(valor_entrada)

for i in listinha:
       resultado = int(valor_entrada/i)
       print('{} nota(s) de R$ {},00'.format(resultado,i))
       valor_entrada = valor_entrada - resultado*i      

File: uri/testing/1018.cpp
#include <iostream>
#include <vector>
using namespace std;

int main(){

       vector<int> listinha = {100,50,20,10,5,2,1};
       int valor_entrada;
       int result_inicial;

       cin >> valor_entrada;

       for(auto i=listinha.begin(); i!=listinha.end(); ++i){

              result_inicial = *i/valor_entrada;
              cout << result_inicial << endl;
              valor_entrada = valor_entrada - (*i*result_inicial);
       }

}